portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -17,55 +17,56 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.common.collect.Lists;
 import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
 import java.io.IOException;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that encapsulates the subtraction game. */
 @WebServlet("/comments")
 public final class CommentsServlet extends HttpServlet {
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String numRequest = getParameter(request, "num", "");
+    String numRequest = getRequestParameter(request, "num", "0"); /* defaultValue= 0*/
```

Comment:
Sorry, I guess I wasn't clear before. It would look like:
```java
String numRequest = getRequestParameter(request, "num", /* defaultValue= */ "0");
```

Reply:
oh ok got it